sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample7_ExportProject.md

```diff
@@ -29,8 +29,8 @@ string projectName = "project-to-export";
 Operation<BinaryData> exportOperation = client.ExportProject(WaitUntil.Completed, projectName);
 
 // Get the resultUrl from the response, which contains the exported project.
-using JsonDocument doc = JsonDocument.Parse(exportOperation.Value.ToStream());
-string resultUrl = doc.RootElement.GetProperty("resultUrl").GetString();
+dynamic result = exportOperation.Value.ToDynamicFromJson();
+string resultUrl = result.resultUrl;
```

Comment:
```suggestion
Uri resultUrl = (Uri)result.resultUrl;
```

Reply:
Turns out the explicit cast isn't needed, or is forced by the compiler anyway.